pyaerocom/io/helpers.py

```diff
@@ -304,6 +304,59 @@ def get_all_names():
             raise Exception("Failed to access model IDs")
     return names
 
+def get_country_name_from_iso(iso_code=None,
+                              filename='country_codes.json',
+                              return_as_dict=False):
+    """get the country name from the 2 digit iso country code
+
+    the underlaying json file was taken from this github repository
+    https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes
+
+    Parameters
+    ----------
+    iso_code : :obj:`str`
+        string containing the 2 character iso code of the country (e.g. no for Norway)
+    filename : :obj:`str` , optional
+        optional string with the json file to read
+    return_as_dict : :obj:`bool`, optional
+        flag to get the entire list of countries as a dictionary with the country codes
+        as keys and the country names as value
+        Useful if you have to get the names for a lot of country codes
+
+    Returns
```

Comment:
I need the dict, otherwise I will read in the json file 4500 times the number of years. A big waste of time.
Returning a dict for a singe entry is also a ware of time. I could make two methods though, but is that really worth it?

Reply:
I think we can leave it this way, since this is a quite special method.